### PR TITLE
Adding quota plugin to new dav endpoint

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
@SergioBertolinSG we might want to think about an integration test for this one - THX

File: build/integration/features/dav-v2.feature [hunk 1]
```diff
@@ -53,3 +53,21 @@ Feature: dav-v2
 		Given Logging in using web as "admin"
 		When Sending a "PROPFIND" to "/remote.php/dav/files/admin/welcome.txt" with requesttoken
 		Then the HTTP status code should be "207"
+
+	Scenario: Uploading a file as recipient using webdav new endpoint having quota
```
Comment (DeepDiver1975) [inline build/integration/features/dav-v2.feature]: while this scenario is still valid - it does not match this pr.

The scenario should look like:
- user0 has 0B quota
- upload of file with 10B fails with 207

Comment (SergioBertolinSG) [inline build/integration/features/dav-v2.feature]: 507 is the expected code isn't?

**Comment by DeepDiver1975** [comment]
thx @SergioBertolinSG

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**